mapbox/libjava-services/src/main/java/com/mapbox/services/api/directions/v5/MapboxDirections.java

```diff
@@ -148,12 +148,24 @@ public Builder() {
       // Set defaults
       this.user = DirectionsCriteria.PROFILE_DEFAULT_USER;
 
-      // We only support polyline encoded geometries to reduce the size of the response.
-      // If we need the corresponding LineString object, this SDK can do the decoding with
-      // LineString.fromPolyline(String polyline, int precision).
+      // by defauly the geometry is polyline with precision 5.
       this.geometries = DirectionsCriteria.GEOMETRY_POLYLINE;
```

Comment:
@cammace Let's default to precision 6. That's what I do on MapMatching and the payload isn't much bigger but there's a gain in precision.